UEBPrep/app/models/playlist.rb

```diff
@@ -1,5 +1,6 @@
 class Playlist < ActiveRecord::Base
   validates :name, presence: true
+  validate :user_id, presence: true
```

Comment:
I think I did it as a has_many rel from the user model, I have definitely been able to reference the playlists from the user. Will get back to you on this one.